gateways/custom/aws-sns/start.go

```diff
@@ -0,0 +1,227 @@
+/*
+Copyright 2018 BlackRock, Inc.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+	http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package aws_sns
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"sync"
+
+	"github.com/AdRoll/goamz/sns"
```

Comment:
why are you choosing this versus the aws-sdk? see: https://github.com/aws/aws-sdk-go

Reply:
For the gateway we just need the structs that for receiving SNS and SQS notifications, so I added `github.com/AdRoll/goamz/sns` cause its simple to use.